### ISSUE TITLE
Facilitate Navigation from Cookie to Corresponding Network Request

### ISSUE THREAD

**Issue by gagan0123** [GoogleChromeLabs/ps-analysis-tool#320]
## Feature Request

**Description:**
This feature aims to enhance the user experience by providing a seamless way to navigate from a cookie listed in the PSAT Extension to its corresponding network request in the network panel. The process would be facilitated by generating a query string from the selected cookie and its domain, which can then be copied to the clipboard for easy pasting in the Network tab.

**Motivation:**
The primary motivation for this feature is to streamline the workflow for users who need to track how specific cookies are used in the network requests. By simplifying this process, users can more efficiently analyze and debug web applications, leading to faster debugging of cookies in the Network tab of Chrome Dev Tools.

**User Story:**
As a web engineer, I want to quickly navigate from a selected cookie in the Cookie table of PSAT Extension tab to its corresponding network request, so that I can efficiently analyze how the cookie is being used in the network activity of my web application.

**Acceptance Criteria:**
- [x] The feature should allow users to right-click on a cookie in the Cookie table of PSAT Extension and select an option to copy the query string based on the selected cookie.
- [x] A query string based on the selected cookie should be automatically generated and copied to the clipboard.
- [x] Users should be able to paste the copied query string in the Network tab to filter and identify the relevant network request.

**Additional Information:**
Example query string:
```
cookie-domain:accounts.google.com cookie-name:__Host-1PLSID 
```

**Comment by kjnanda** [comment]
@amovar18 The added functionality looks good to me. but I have some minor UI feedbacks to share which can be addressed later. 

- [ ] **The `Copy network filter string.` is not aligned in the centre.**   

<kbd>![image](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/15173772/7962a7fe-51b5-4d01-95da-ae53353f74e0)</kbd>

----

- [ ] **Also, I feel there is slightly different color in bg wrt Application Tab.**

| Application Tab | PSAT |
|---- |---- |
| ![image](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/15173772/a2174161-15be-4fbc-a145-d4ab53e4a609) | ![image](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/15173772/90002b88-06ba-44f1-8935-378b2bf8d374) |

**Comment by ayushnirwal** [comment]
>Also, I feel there is slightly different color in bg wrt Application Tab

Thanks @kjnanda for pointing this out let's make an enhancement issue for this the context menu used everywhere else is styled the same.

**Comment by maitreyie-chavan** [comment]
Released as a part of [v0.4.0](https://github.com/GoogleChromeLabs/ps-analysis-tool/releases/tag/v0.4.0), closing this issue.
